--- src/Www.php	2020-12-02 01:07:04.000000000 +0100
+++ ../../../patch/middlewares/www/src/Www.php	2022-05-24 17:52:45.931219877 +0200
@@ -49,7 +49,7 @@
 
         if ($uri->getHost() !== $host) {
             return $this->responseFactory->createResponse(301)
-                ->withHeader('Location', (string) $uri->withHost($host));
+                ->withHeader('Location', (string) $uri->withPort(null)->withHost($host));
         }
 
         return $handler->handle($request);
