--- src/Rule/AbstractDateTime.php	2023-07-27 14:28:28.075769073 +0000
+++ ../../../patch/aura/filter/src/Rule/AbstractDateTime.php	2023-07-27 14:47:15.956518437 +0000
@@ -48,7 +48,7 @@
 
         // invalid dates (like 1979-02-29) show up as warnings.
         $errors = DateTime::getLastErrors();
-        if ($errors['warnings']) {
+        if (is_array($errors) && $errors['warnings']) {
             return false;
         }
 
